README.md

```diff
@@ -1,17 +1,24 @@
-## How to run 'mpi_program.f90'
+## How to run 'cart_sub.f90'
```

Comment:
Is this `cart_sub.f90` at the root directory? If not, maybe `tests/cart_sub.f90`?